### ISSUE TITLE
end2endtest_staging job is failing every time; atServers are not being created

### ISSUE THREAD

**Issue by gkc** [atsign-foundation/at_server#1474]
### Describe the bug

- new atSigns are being registered successfully, but something is subsequently failing - for example [here](https://github.com/atsign-foundation/at_server/actions/runs/5667036592/job/15355436609) two new atSigns were created:
```
AT_SIGN_1: test7cd18781_5943_4a8d_a6f1_8c66a9f2031e1
AT_SIGN_2: test3f01bc5d_14b3_4c76_b1ab_e576553497bb1
```
- a manual lookup of those atSigns on `root.atsign.wtf:64` shows that the atServers were indeed assigned a hostname
```
@test7cd18781_5943_4a8d_a6f1_8c66a9f2031e1
e91ab171-82e6-5125-9186-ff1cbff78ee9.staging0001.atsign.zone:1472
@test3f01bc5d_14b3_4c76_b1ab_e576553497bb1
33608b39-d296-5163-b359-347552f94df7.staging0001.atsign.zone:1473
```
- **however** neither of those addresses is reachable
```
gary@gkc2019-2 ~ % openssl s_client -connect e91ab171-82e6-5125-9186-ff1cbff78ee9.staging0001.atsign.zone:1472
getaddrinfo: nodename nor servname provided, or not known
connect:errno=0
gary@gkc2019-2 ~ % openssl s_client -connect 33608b39-d296-5163-b359-347552f94df7.staging0001.atsign.zone:1473
getaddrinfo: nodename nor servname provided, or not known
connect:errno=0
gary@gkc2019-2 ~ % 
```

### Steps to reproduce

Multiple actions runs have failed, a link is provided above

### Expected behavior

Should not fail.

**Comment by cpswan** [comment]
@gkc Looking at a [recent run](https://github.com/atsign-foundation/at_server/actions/runs/5678587078/job/15389378282) I see `Error fetching Atsign Hostname`

If I try to get the name myself then it's in the root:

```
(echo test815d3141_4719_4333_9cd5_efb321df9e571; sleep 1) | openssl s_client -brief -connect root.atsign.wtf:64 2> /dev/null | grep --color=none "^@.*:" | cut -d'@' -f2
2a34e09e-5f04-54d2-b1da-ba35ac0def96.staging0001.atsign.zone:1489
```

NB I'm using `-brief` there, which isn't in the prod script (and should be)
NB2 The prod script should have retry logic rather than just failing on first try.

Once we get past the root lookup we're back to the problem that the atServer doesn't exist on the staging environment. So the creation process has got to creating the root entry, but hasn't actually spun up an atServer on the Swarm.

Although the script in the workflow doesn't use SSH. I suspect that the provisioning process does, and is presently blocked due to the SSH lockdown we've done in the past couple of days. @kumarnarendra701 @athandle we need to get the IP(s) that those SSH connections come from allow listed ASAP. Meanwhile I'm going to revert the changes so that provisioning works.

**Comment by cpswan** [comment]
Tests are now passing again.

**Comment by kumarnarendra701** [comment]
@cpswan - We need to allow SSH access from our Kubernetes cluster to the staging swarm cluster. Here's the IP that needs to be whitelisted.

34.66.89.205 
34.72.242.19 
34.66.64.191

**Comment by cpswan** [comment]
@kumarnarendra701 can you please confirm if those IPs apply to staging or prod or both?

**Comment by kumarnarendra701** [comment]
@cpswan - The above IPs are for staging only and please let me know if you need prod IPs.

**Comment by cpswan** [comment]
@kumarnarendra701 can you please let me know the prod IPs too?

**Comment by kumarnarendra701** [comment]
@cpswan Prod IP - 35.202.205.184(Our prod servers are running on private subnet so only one IP of NAT)

**Comment by cpswan** [comment]
Reopening this as tests are failing again even though atServers are being created.

I'm going to introduce some retry logic to the script so it doesn't just fail if the atServers don't come up quickly enough.

**Comment by cpswan** [comment]
Looks like we have cases where the atServer service is created on the Swarm, but it's unreachable on its UUID.staging0001.atsign.zone:port because there's no CNAME for UUID.staging0001.atsign.zone to staging0001.lb.atsign.zone

**Comment by cpswan** [comment]
Confirmed a problem with DigitalOcean:

`Error: Unable to initialize DigitalOcean API client: access token is required. (hint: run 'doctl auth init')`

Here's the full log for one of the atServers:

```
Secondary creation started.
testb51155ab_8406_411a_9fe2_afc5a9dc83a51
1500
48ce8dd5-1f19-52a7-8649-a554447d1a8c
d9/4d/76/dd/d94d76dd121e07fe9834327ce7ab432e
df7471cf6c3cdd4310dea21faa6af6d7f0f154463e583c8ccb35cbf118bdb8dd5b5e49417da4bf3273fefccb8fb73a6791a77e71ec47b327ce08a92f70533ae8
staging0001
/etc/letsencrypt/live/48ce8dd5-1f19-52a7-8649-a554447d1a8c.staging0001.atsign.zone
notAfter=Oct 25 11:10:09 2023 GMT
Certificate will not expire
Given certificate is valid.
Error: Unable to initialize DigitalOcean API client: access token is required. (hint: run 'doctl auth init')
Creating service 48ce8dd5-1f19-52a7-8649-a554447d1a8c_secondary
Secondary creation completed.
--------------------------------
```

**Comment by cpswan** [comment]
I've just done a `doctl auth init`

**Comment by cpswan** [comment]
Closed by #1481
